Events.php

```diff
@@ -19,6 +20,10 @@ class Events
 
     public static function onFileHandlerCollection($event)
     {
+        if (!Yii::$app->user->can(CanUseOnlyOffice::class)) {
+            //return;
```

Comment:
This probably shouldn't be commented out, should it?

Reply:
Oh yes, sorry, fixed in https://github.com/ONLYOFFICE/onlyoffice-humhub/pull/42/commits/05f5e89f4fc7fceceb5a09ca72eb9e3024b51c6e